src/confcom/azext_confcom/container.py

```diff
@@ -321,7 +282,7 @@ def extract_user(container_json: Any) -> Dict:
     return user
 
 
-def extract_capabilities(container_json: Any, allow_elevated: bool):
```

Comment:
you're duplicating the code for getting the privileged value here and in the extract_allow_elevated function, I'd just pass the allow_elevated value into the function here and remove the duplicated code